ankerctl.py

```diff
@@ -503,12 +503,14 @@ def app_api_files_local():
 @webserver.command("run", help="Run ankerctl webserver")
 @click.option("--host", default='127.0.0.1', envvar="FLASK_HOST", help="Network interface to bind to")
 @click.option("--port", default=4470, envvar="FLASK_PORT", help="Port to bind to")
+@click.option("--no-cache", "-nc", is_flag=True, help="Disbale the template cache")
```

Comment:
This seems like a developer-only option. I'd rather make this an environment variable, as there's very little value for the average user in having this option.

Reply:
Also, specifying multi-letter options with a single dash is bad style - if it even works?